cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -50,6 +50,12 @@ var PoolAddEventHandled = false
 // PoolNamePrefix is a typed string to store pool name prefix
 type PoolNamePrefix string
 
+// CStorPools is a map of string, API version of CStorPool
+var CStorPools map[string]*apis.CStorPool
```

Comment:
Can we rename this to `ImportedCStorPools` ?
By looking at code, above name was my best guess.
In addition, we need not talk about language semantics like map of string etc unless its not essential to the behaviour.
e.g. we can write the comment as `CStorPools is a map of imported cstor pools identified via their UIDs`

Reply:
done